server/cmwell-cons/src/main/scala/ctl.scala

```diff
@@ -315,9 +316,9 @@ abstract class Host(user: String,
     // scalastyle:on
   }
 
-  def ips = ipMappings.getIps
+  def ips = hostIps.toList
```

Comment:
can we remove this method and only use the member?

Reply:
no